composer.json

```diff
@@ -20,14 +20,18 @@
         "friendsofphp/php-cs-fixer": "^3.4",
         "khanamiryan/qrcode-detector-decoder": "^1.0.3",
         "phpstan/phpstan": "^1.2",
+        "setasign/fpdi": "^2.3.6",
+        "setasign/fpdf": "^1.8.4",
```

Comment:
Is `setasign/fpdf` needed if there is already `fpdf/fpdf`?

Reply:
Unfortunately yes, because `setasign/fpdf` doesn't extend `fpdf/fpdf` it just implements the same properties and methods so it can easily be switched between the two of them.